### ISSUE TITLE
Improve web app performance and accessibility

### ISSUE THREAD

**Issue by abdultolba** [abdultolba/social-mern#13]
Currently, friend.ly isn't as accessible as it should be and does not conform completely with [WCAG 2.0 guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/).

The [lighthouse report](https://googlechrome.github.io/lighthouse/viewer/?gist=646b206e4b1304ae76eb88918b8390e1) generated gives scores of _82_, _82_, and _77_ for _performance_, _accessibility_, and _best practices_, respectively.
**Note:** Some of the console warnings are due to ad blocker extensions and can be ignored.

@abdultolba, please take the time to go through it and work on improvements 🤙
